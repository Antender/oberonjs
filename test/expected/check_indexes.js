<rtl code>
var m = function (){

function charArray(a/*ARRAY OF CHAR*/){
	var c = 0;
	c = RTL$.charAt(a, 1);
	c = RTL$.charAt(a, RTL$.charAt(a, 1));
}
}();
