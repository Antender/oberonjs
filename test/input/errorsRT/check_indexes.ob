(*options:{"checkIndexes": true}*)
MODULE m;

PROCEDURE charArray(a: ARRAY OF CHAR);
VAR
	c: CHAR;
BEGIN
	c := a[0];
END charArray;

BEGIN
	charArray("");
END m.