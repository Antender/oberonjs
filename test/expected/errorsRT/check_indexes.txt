Error: index out of bounds: 0